pytorch/inference/docker/1.3.1/py3/Dockerfile.eia

```diff
@@ -18,7 +18,10 @@ ENV SAGEMAKER_SERVING_MODULE sagemaker_pytorch_serving_container.serving:main
 ENV TEMP=/home/model-server/tmp
 
 RUN apt-get update \
- && apt-get install -y --allow-downgrades --allow-change-held-packages --no-install-recommends \
+    && apt-get install -y --no-install-recommends software-properties-common \
+    && add-apt-repository ppa:openjdk-r/ppa \
```

Comment:
Is installing via ppa necessary? Does TS need jdk11 [if there are breaking changes between jdk8 and jdk11, could you comment on that]? Also, what does the software-properties-common package bring in? 


@TusharKanekiDey Could you check for size increase in the package as jdk updates from jdk8-jdk11?

Reply:
I think these lines are added to allow the installation of jdk11(https://stackoverflow.com/questions/52504825/how-to-install-jdk-11-under-ubuntu)

Reply:
> Does TS need jdk11  
Yes.  https://github.com/pytorch/serve#install-with-pip. 
I am not aware of any breaking changes.

Also, what does the software-properties-common package bring in?
```add-apt-repository``` is a part of software-properties-common